src/textual/_xterm_parser.py

```diff
@@ -31,6 +31,10 @@
 """Sequence received when the terminal receives focus."""
 FOCUSOUT: Final[str] = "\x1b[O"
 """Sequence received when focus is lost from the terminal."""
+BG_COLOR: Final[str] = "\x1b]11;rgb:"
```

Comment:
We have a policy in Textual of no abbreviations, so BG should be BACKGROUND.

I think this would be better done as a regex.

Do you have a reference to the docs? Can you be certain it is sent in a two hex characters?

Reply:
I will fix the abbreviation. Here is a link to some documentation on the escape sequence:

https://www.xfree86.org/current/ctlseqs.html

Search for "Operating System Controls"